supporting_content/metrics/pricingquantity.md

```diff
@@ -0,0 +1,87 @@
+# Column: PricingQuantity
+
+## Example treatment for non-metered charges (e.g. Tax Treatment)
+
+Example A
+| Description          | Pricing Quantity          | Unit Price   | Total     | 
+|:---------------------|:--------------------------|:-------------|-----------|
+| Compute Usage        | 100                       | 1.0          | $100      |
+| Tax on Compute Usage | 100                       | 0.2          | $20       |
+| TOTAL                | 200                       | 1.2          | $120      |
+NOTE: Double counting of hours
+
+Example B
+| Description          | Pricing Quantity          | Unit Price   | Total     | 
+|:---------------------|:--------------------------|:-------------|-----------|
+| Compute Usage        | 100                       | 1.0          | $100      |
+| Tax on Compute Usage | 1                         | 20           | $20       |
+| TOTAL                | 101                       | 21           | $120      |
+NOTE: Skews average of Unit price
+
+Example C
+| Description          | Pricing Quantity          | Unit Price   | Total     | 
+|:---------------------|:--------------------------|:-------------|-----------|
+| Compute Usage        | 100                       | 1.0          | $100      |
+| Tax on Compute Usage | NULL                      | NULL         | $20       |
+| TOTAL                | 100                       | 1.0          | $120      |
+NOTE: Preferred option?
+
+## Example provider mappings
+
+Current column mappings found in available data sets:
+
+| Provider  | Data set                     | Column                   |
+|:----------|:-----------------------------|:-------------------------|
+| AWS       | Cost and Usage Report        | lineItem/UsageAmount (NOTE: how to handle reservation/TotalReservedUnits|
+| GCP       | BigQuery Billing             | usage.amount_in_pricing_units (NOTE: usage.amount provides QuantityInUsageUnit) |
```

Comment:
If the naming convention has indeed changed from `Quantity in * Unit` to `* Quantity`, then the docs in this PR will need a scrub to replace any existing use of the old convention, such as in this table here.

Reply:
resolved. thanks for pointing this out :)